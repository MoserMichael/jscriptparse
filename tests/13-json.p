    # conversion of variable to string and vice versa

    dct = { "persons": { "id": "323412343123", "name": "Michael", "surname": "Moser", "age": 52 }, "stuff": [3, 2, 1] }
    js = toJsonString( dct )
    println( "json: {js}" )
    
    vl = parseJsonString(js)
    vl['persons']['id'] = 123
    vl['persons']['age'] = 22
    js = toJsonString(vl)
    println( "json: {js}" )

