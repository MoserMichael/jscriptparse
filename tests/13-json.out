json: {"persons":{"id":"323412343123","name":"Michael","surname":"Moser","age":52},"stuff":[3,2,1]}
json: {"persons":{"id":123,"name":"Michael","surname":"Moser","age":22},"stuff":[3,2,1]}
